scripts/ValidateProjectVersionUpdated.ps1

```diff
@@ -29,9 +29,9 @@ Param(
 [xml]$xmlDoc = Get-Content $projectPath
 
 # Assumption: There is only one PropertyGroup
-$versionPrefixString = $xmlDoc.Project.PropertyGroup.VersionPrefix
-if($xmlDoc.Project.PropertyGroup.VersionSuffix){
-    $versionPrefixString = $versionPrefixString + "-"  + $xmlDoc.Project.PropertyGroup.VersionSuffix
+$versionPrefixString = $xmlDoc.Project.PropertyGroup.VersionPrefix[0]
```

Comment:
I think this is happening because of two PropertyGroups, we should select the one with the project's details with a Where statement.